ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/route53/private-hosted-zone/deprovision/cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-deprovision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-deprovision
+  steps:
+  - ref: gather-aws-console
+  - chain: gather
+  - ref: ipi-deprovision-deprovision
+  - ref: aws-deprovision-stacks
+  - ref: aws-deprovision-s3buckets
```

Comment:
per https://github.com/openshift/release/pull/27656/files#r851073168, this step is also not required.

Reply:
Done